.github/workflows/test-phpunit.yml

```
@@ -126,7 +126,15 @@ jobs:
         if: matrix.php-versions == '8.0'
         run: echo "TACHYCARDIA_MONITOR_GA=enabled" >> $GITHUB_ENV
 
-      - name: Test with PHPUnit
+      - name: Test with PHPUnit w/o Coverage
+        if: matrix.php-versions != '8.0'
+        run: script -e -c "vendor/bin/phpunit --color=always --exclude-group=auto-review --no-coverage"
+        env:
+          DB: ${{ matrix.db-platforms }}
+          TERM: xterm-256color
+
+      - name: Test with PHPUnit with Coverage
+        if: matrix.php-versions == '8.0'
         run: script -e -c "vendor/bin/phpunit --color=always --exclude-group=auto-review"
```

I think it may be better to use GHA flags instead of duplicating. Something like:
```suggestion
      - name: Compute coverage option
        uses: actions/github-script@v3
        id: phpunit-coverage-option
        with:
          script: 'return "${{ matrix.php-versions }}" == "8.0" ? "--no-coverage" : ""'
          result-encoding: string

      - name: Test with PHPUnit
        run: script -e -c "vendor/bin/phpunit --color=always --exclude-group=auto-review ${{ steps.phpunit-coverage-option.outputs.result }}"
```